src/games/strategy/engine/framework/mapDownload/DownloadRunnable.java

```diff
@@ -52,26 +59,45 @@ public void setError(final String error) {
 
   @Override
   public void run() {
```

Comment:
New fork in functionality here allows us to read triplea_maps.xml from local file or downloading from net. Breaks the design of DownloadRunnable a bit, but in terms of making a focused/non-invasive change, this was a compromise to avoid changing things around in a much more radical way.